pizzakit-plugin/includes/pizzakit.php

```diff
@@ -218,7 +215,7 @@ private static function insert_into_tables($_data)
 
 		$json = file_get_contents(plugin_dir_path(__FILE__) . 'items_for_sale.json');
 		$decoded = json_decode($json, true);
-		$items = array_merge($decoded['main_items'],$decoded['extras']);
+		$items = $decoded['menu'];
 
```

Comment:
Is this because all items are now stored in 'menu', instead of being separated into two arrays?

The rest looks good

Reply:
Yep